app.json

```diff
@@ -11,6 +11,10 @@
       "description": "A Slack API token for your account",
       "value": "SLACK API TOKEN GOES HERE"
     },
+    "SLACK_CHANNEL": {
+      "description": "The Slack channel in which you'd like your notables to appear",
+      "value": "SLACK CHANNEL NAME HERE"
```

Comment:
I'd put "SLACK CHANNEL NAME GOES HERE" just to make it consistent with the other values! :smile: